gulpfile.js

```diff
@@ -241,8 +241,25 @@ function endBuildStep(stepName, targetName, startTime) {
  * @return {!Promise}
  */
 function buildExtensions(options) {
-  var results = [];
+  const results = [];
+  let extensionsToBuild = [];
+  // --noextensions flag skip building extensions
+  if (!!argv.noextensions) {
+    return Promise.all(results);
```

Comment:
Move this to the top and `return Promise.resolve();`

Reply:
done